client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -1,10 +1,11 @@
+/*eslint-disable*/
```

Comment:
Er, apologies. Can't believe I missed that on the pre-commit diff. 
I only put that in there to allow `debugger` while developing. Will push a fix to this branch.